javascript/answers/1709abhishek-q7.html

```diff
@@ -0,0 +1,65 @@
+<html>
+    <head>
+
+        <!-- Global site tag (gtag.js) - Google Analytics -->
+        <script async src="https://www.googletagmanager.com/gtag/js?id=UA-157837769-1"></script>
+        <script>
+        window.dataLayer = window.dataLayer || [];
+        function gtag(){dataLayer.push(arguments);}
+        gtag('js', new Date());
+        gtag('config', 'UA-157837769-1');
+        </script>
+
+        <title>
+            JavaScript | InterQues | Coding Ninjas
+        </title>
+
+        <link rel="shortcut icon" type="image/png" href="../../assets/favicon.png"/>
+
+        <meta name="description" content="What is the difference between == and ===?
+">
+    </head>
+    <body>
+
+        <div>
+            <img src="../../assets/CN_logo.png" height='72px' alt="Coding Ninjas Logo">
+        </div>
+
+        <div>
+            <p>
+                <small>
+
+                    <a href="../../index.html">Home</a>
+                    &gt; <a href="../index.html">JavaScript Questions</a>
+                    &gt; What is the difference between == and ===?
+
+
+                </small>
+
+            </p>
+        </div>
+
+
+        <div>
+            <div>
+                <h4>
+                  What is the difference between == and ===?
+                </h4>
+            </div>
+            <div>
+                <br>
+                <strong>Answer:</strong>
+                <p> The == operator checks equality only whereas === checks equality, and data type, i.e., a value must be of the same type.</p>
```

Comment:
Please give an example to support your answer. :)

Reply:
<img width="638" alt="Screenshot 2020-02-17 at 1 14 44 PM" src="https://user-images.githubusercontent.com/30382947/74633381-88749d00-5187-11ea-8667-757708efd4e1.png">